app/model/RuleContext.scala

```diff
@@ -47,7 +53,7 @@ case class RuleContext(authContext: AuthContext)(implicit hc: HeaderCarrier) {
 
   lazy val currentCoAFEAuthority = frontendAuthConnector.currentCoAFEAuthority()
 
-  lazy val internalUserIdentifier = currentCoAFEAuthority.map(_.internalUserIdentifier)
+  lazy val internalUserIdentifier = currentCoAFEAuthority.map(authority => authority.internalUserIdentifier)
```

Comment:
connector call could go here. I already have this change on my branch:
https://github.com/hmrc/tax-account-router-frontend/blob/TSV-305/app/model/RuleContext.scala#L54